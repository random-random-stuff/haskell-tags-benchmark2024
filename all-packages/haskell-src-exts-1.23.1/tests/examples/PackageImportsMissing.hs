module PackageImportsMissing where

import "wibble" PackageImports.Default
import "wibble" PackageImports.Builtin.All
