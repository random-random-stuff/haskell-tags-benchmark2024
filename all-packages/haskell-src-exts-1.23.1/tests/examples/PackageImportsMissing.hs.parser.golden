ParseFailed
  (SrcLoc "tests/examples/PackageImportsMissing.hs" 3 17)
  "PackageImports language extension is not enabled. Please add {-# LANGUAGE PackageImports #-} pragma at the top of your module."
